buildall.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
```

Comment:
I removed the large Dockerfile and replaced it with one that enables us to control which NGINX versions are built for which operating system. Eg. Ubuntu 24 is new in R32 (1.25.5) and should not be built for earlier versions.